/**:
  ros__parameters:
    # model: OSI-64
    # ip_addr: 192.168.1.2
    # lidar_port: 7502

    beam_azimuth_angles: [3.11, 0.9399999999999999, -1.22, -3.37, 3.08, 0.93, -1.19, -3.32, 3.07, 0.95, -1.17, -3.28, 3.06, 0.97, -1.14, -3.24, 3.07, 0.97, -1.12, -3.21, 3.07, 0.99, -1.1, -3.18, 3.08, 1, -1.08, -3.16, 3.1, 1.02, -1.06, -3.14, 3.11, 1.03, -1.04, -3.12, 3.14, 1.05, -1.03, -3.1, 3.16, 1.07, -1.02, -3.11, 3.18, 1.1, -0.99, -3.09, 3.22, 1.12, -0.98, -3.1, 3.26, 1.14, -0.96, -3.1, 3.3, 1.17, -0.96, -3.1, 3.36, 1.22, -0.95, -3.12]
    beam_altitude_angles: [16.83, 16.22, 15.65, 15.11, 14.62, 14.04, 13.49, 12.95, 12.47, 11.9, 11.35, 10.82, 10.35, 9.789999999999999, 9.24, 8.710000000000001, 8.24, 7.69, 7.14, 6.61, 6.15, 5.6, 5.06, 4.52, 4.06, 3.51, 2.97, 2.43, 1.96, 1.42, 0.88, 0.34, -0.13, -0.67, -1.21, -1.76, -2.22, -2.76, -3.31, -3.85, -4.32, -4.86, -5.4, -5.95, -6.41, -6.96, -7.49, -8.039999999999999, -8.52, -9.050000000000001, -9.6, -10.15, -10.63, -11.17, -11.71, -12.28, -12.76, -13.29, -13.85, -14.43, -14.93, -15.47, -16.03, -16.64]
    lidar_origin_to_beam_origin_mm: 12.163
    lidar_to_sensor_transform: [-1.0,0.0,0.0,0.0, 0.0,-1.0,0.0,0.0, 0.0,0.0,1.0,36.180, 0.0,0.0,0.0,1.0]
